noggin/controller/authentication.py

```diff
@@ -65,34 +64,31 @@ def login():
 def otp_sync():
     form = SyncTokenForm()
     if form.validate_on_submit():
-        try:
-            ipa = untouched_ipa_client(app)
-            response = ipa.otptoken_sync(
-                user=form.username.data,
-                password=form.password.data,
-                first_code=form.first_code.data,
-                second_code=form.second_code.data,
-                token=form.token.data,
-            )
-
-            if (
-                response.status_code == 200
-                and "Token sync rejected" not in response.text
-            ):
-                flash('Token successfully synchronized', category='success')
-            else:
-                flash(
-                    'Something went wrong trying to sync OTP token.', category='danger'
+        with handle_form_errors(form):
```

Comment:
Yeah I guess you can use `handle_form_errors` here, it's not entirely necessary because you only have one set of exception handlers, but it'll work.
If the reason for the existence of `handle_form_errors` is not clear, please say so and I'll improve the documentation (which is severely lacking).

Reply:
maybe I do misunderstand then, because when I don't use it raising the `FormError` actually raises an error and the application shows an exception as its not caught anywhere now.